nipype/pipeline/engine/nodes.py

```diff
@@ -1260,7 +1260,7 @@ def _run_interface(self, execute=True, updatehash=False):
                 stop_first=str2bool(
                     self.config['execution']['stop_on_first_crash'])))
         # And store results
-        _save_resultfile(result, cwd, self.name)
+        _save_resultfile(result, cwd, self.name, rebase=False)
```

Comment:
Should this be `self.config.getboolean('execution', 'use_relative_paths')`?

```suggestion
        _save_resultfile(result, cwd, self.name,
                         rebase=self.config.getboolean('execution', 'use_relative_paths'))
```

Reply:
This is the discussion point 2 I wanted to bring up atop. On the one hand, should we expose the inner workings of the engine - after all, adding that here does not change anything to the user (just that they won't be able to move the work directory anymore).

On the other hand, for this to be completely consistent across the node implementation, we need to also rebase/resolve the inputs pickle (actually I'll open an issue because this should be addressed).

I'm under the impression that `use_relative_paths` could be useful for the user at the interface level, forcing interfaces to return relative paths if desired.

Reply:
@satra can you share your views as regards `use_relative_paths`

@effigies WDYT about not exposing the inner workings (paths) of the results file to the users?

Reply:
My overall perspective is "Let's not change the API if we don't absolutely have to to fix the bug." Which includes config file options.

I may not be understanding your position though, so we might be talking past each other.

Reply:
Okay, [from that standpoint](https://nipype.readthedocs.io/en/1.1.1/users/config_file.html):

> **use_relative_paths**
> Should the paths stored in results (and used to look for inputs) be relative or absolute. Relative paths allow moving the whole working directory around but may cause problems with symlinks. (possible values: true and false; default value: false)

The option is clearly defined specifically for the results file. Under that perspective, yes, any rebasing should be done only if `use_relative_paths` is true.

Reply:
But actually not at the point you suggested, because that one happens at MapNode aggregation.

Reply:
Oh, okay. Sorry, my attention is split a lot of ways right now.

Reply:
Done, however, we may want to add parameterized tests with ``use_relative_paths`` on and off.